api.md

```diff
@@ -1876,6 +1877,93 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_job() :community_function: [](add_job)
+
+Register a custom procedure or function to be scheduled by our automation framework.
+Please read the [instructions][using-custom-jobs] for more details including
+multiple example jobs.
+
+#### Required Arguments [](add_job-required-arguments)
+
+|Name|Description|
+|---|---|
+| `proc` | (REGPROC) Name of the function or procedure to register as job|
+| `schedule_interval` | (INTERVAL) Interval between executions of this job|
```

Comment:
If it is not a scheduled job, wouldn't this be NULL? Shouldn't this be optional in that case?

Reply:
the column is not null so it is required, it's in the design doc